### PR TITLE
Dom api task

### PR DESCRIPTION
**Pull request by AndewTu**
_Replace task name and links as appropriate and remove this line._

## DOM-practice-task

[Demo](https://andewtu.github.io/kottans-frontend/) | [Codebase](https://github.com/AndewTu/frontend-2019-p2p/tree/dom-api-task/submissions/AndewTu/js-dom)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by AndewTu** [comment]
Hi!
@Roman-Halenko 
Thank you for paying attention to this. I fixed it!)

**Comment by Roman-Halenko** [comment]
@AndewTu, please, always sync the changes you make here to your task repo. Since yet your demo doesn't look like changes are implemented.

**Comment by AndewTu** [comment]
@Roman-Halenko update demo, thank you again)

File: submissions/AndewTu/js-dom/app.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+	.btn#marshall{color: #EE2D25; border: 2px #EE2D25 solid;}
+	.btn#chase{color: #3D4969; border: 2px #3D4969 solid;}
+	.btn#skye{color: #C45376; border: 2px #C45376 solid;}
+	.btn#rubble{color: #D7BE28; border: 2px #D7BE28 solid;}
+	.btn#rocky{color: #6AB950; border: 2px #6AB950 solid;}
+	.btn#zuma{color: #F47736; border: 2px #F47736 solid;}
+	.btn#everest{color: #5AC2B3; border: 2px #5AC2B3 solid;}
+	.btn#marshall:hover, .btn#marshall.active{background-color: #EE2D25;transition: background-color 1s; color: #fff; outline:0 !important;}
+	.btn#chase:hover, .btn#chase.active{background-color: #3D4969;transition: background-color 1s; color: #fff; outline:0 !important;}
+	.btn#skye:hover, .btn#skye.active{background-color: #C45376;transition: background-color 1s; color: #fff; outline:0 !important;}
```
Comment (Roman-Halenko) [inline submissions/AndewTu/js-dom/app.css]: 1) Don't ever describe selector's rules in a single line.
1) [Selector and Declaration Separation](https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation)
1) Commonly, using `!important` when you just started project may tell us that you chose a wrong way. If I get right the course of your thoughts - now you may rid of all `outline:0 !important;`, and just one time write:

        .btn:focus {
          outline: none;
        }

File: submissions/AndewTu/js-dom/app.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
+
```
[the remark this answers is not in the document]
Comment (AndewTu) [reply submissions/AndewTu/js-dom/app.css]: fix it

File: submissions/AndewTu/js-dom/app.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+body{
```
[the remark this answers is not in the document]
Comment (AndewTu) [reply submissions/AndewTu/js-dom/app.css]: fix it

File: submissions/AndewTu/js-dom/app.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+li{
+	padding: 10px;
+}
+.btn{	
+	height: 45px;
+	width: 200px;
+	font-size: 1em;
+    font-weight: 600;
+	margin:5px;
+    text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (AndewTu) [reply submissions/AndewTu/js-dom/app.css]: fix it

**Comment by AndewTu** [comment]
@Roman-Halenko  thanks for you comments, amended as per edits.

File: submissions/AndewTu/js-dom/index.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+		<div class="puppies">
+				<h1 class="name">Skye</h1>
+				<img class="pic" src="img/skye.png" caption="puppy">
+				<p class="desc">At seven years old, Skye is a cute, smart Cockapoo puppy. She is a fearless daredevil who will try anything with grace and a smile. She’s smart, loyal, and quick with a quip, sometimes gently teasing the bigger dogs who can’t seem to keep up! She loves to get paw-dicures and look good. When there’s no mission for the PAW Patrol, she can be found snowboarding or playing her favorite videogame, Pup Pup Boogie.</p>
+		</div>	
+	</div>
+	</div>
+<script src="app.js"></script>	
+</body>
+</html>
```
Comment (Temu4) [inline submissions/AndewTu/js-dom/index.html]: It's good practice, to leave an empty line after your code. This task can be automated by your IDE.

**Pull request closed without merging**